plugins/GranularPitchShifter/CMakeLists.txt

```
@@ -0,0 +1,3 @@
+INCLUDE(BuildPlugin)
+
+BUILD_PLUGIN(granularpitchshifter GranularPitchShifterEffect.cpp GranularPitchShifterControls.cpp GranularPitchShifterControlDialog.cpp MOCFILES GranularPitchShifterControls.h GranularPitchShifterControlDialog.h EMBEDDED_RESOURCES *.png)
```

Switch to a little more standard CMake style
```suggestion
include(BuildPlugin)

build_plugin(granularpitchshifter
	GranularPitchShifterEffect.cpp
	GranularPitchShifterControls.cpp
	GranularPitchShifterControlDialog.cpp
	MOCFILES GranularPitchShifterControls.h GranularPitchShifterControlDialog.h
	EMBEDDED_RESOURCES *.png)
```